src/main/java/seedu/badmaths/Command.java

```diff
@@ -53,22 +53,6 @@ public boolean isInvalidTodo(String todo) {
         return todo.equals("Invalid todo");
     }
 
-    public boolean isInvalidIndex(int index, NotesList notes) {
-        return (index < 0 || index >= notes.getSize());
-    }
-
-    public boolean isAnInt(String todo) {
-        try {
-            Integer.parseInt(todo);
-        } catch (NumberFormatException numberException) {
-            Ui.printInvalidNumberEntered();
-            return false;
-        }
-        return true;
-    }
-
-
-
```

Comment:
@YC-Michael @0nandon @khooyourun @ZiqiuZeng 

Please check if removing these will affect your functionalities or not as it was reflected to me by Intellij that this 2 functions were not used at all. Hence I removed them.